src/Raven.Server/Documents/ETL/Providers/AI/Embeddings/EmbeddingsGenerationTaskIdentifier.cs

```diff
@@ -0,0 +1,3 @@
+﻿namespace Raven.Server.Documents.ETL.Providers.AI.Embeddings;
+
+public record EmbeddingsGenerationTaskIdentifier(string Value);
```

Comment:
Why does this exists?

Reply:
I actually like that, because then you know what you want exactly in the method and cannot make a mistake that wrong string was passed as a identifier

Reply:
I actually like that, because then you know what you want exactly in the method and cannot make a mistake that wrong string was passed as a identifier

Reply:
Yes, mostly to distinguish between `AiConnectionStringIdentifier` and `EmbeddingsGenerationTaskIdentifier`